Classes/Service/JsonServer.php

```diff
@@ -118,7 +118,7 @@ protected function getServiceUri()
     {
         $uri = GeneralUtility::getIndpEnv('TYPO3_REQUEST_URL');
         $parts = parse_url($uri);
-        $parts['query'] = 'eID=JsonServer&amp;server=' . $this->serverKey;
+        $parts['query'] = 'eID=JsonServer&server=' . $this->serverKey;
```

Comment:
I saw you explicit set this: https://github.com/lochmueller/autoloader/pull/65/commits/fed22a5e17d309d79a6b8681a55eaa2d513e8ec8

So I merge your example extension.... thanks